### PR TITLE
[v0.6.0] Update node bridge

### PR DESCRIPTION
**Pull request by jdkanani**

### CODE REVIEW

**Comment by pedrouid** [comment]
This looks good! But let's separate the Push server from the Bridge server to match the spec.

**Comment by jdkanani** [comment]
@pedrouid removed push server from bridge server! Should I also change `transaction` to `call`?

**Comment by pedrouid** [comment]
> @pedrouid removed push server from bridge server! Should I also change `transaction` to `call`?

Not yet, I've labelled this PR with v0.6.0 so that it's clear that which spec version we are dealing with.

File: src/api.js [hunk 1]
```diff
@@ -1,290 +1,233 @@
...
 
-// get message body for given dapp name
-function getMessageBody(dappName) {
-  return `New request from ${dappName}`;
-}
+import config from './config'
+import * as keystore from './keystore'
 
 //
 // Send webhook
```
Comment (pedrouid) [inline src/api.js]: Send push notification

**Comment by pedrouid** [comment]
@jdkanani check out PR #3 where I've made the changes for renaming transactions to calls

**Pull request merged**